include/glow/IR/GraphScheduler.h

```diff
@@ -0,0 +1,97 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#ifndef GLOW_IR_GRAPH_SCHEDULER_H
+#define GLOW_IR_GRAPH_SCHEDULER_H
+
+#include "glow/Graph/Graph.h"
+#include "glow/Graph/Nodes.h"
+#include "glow/Graph/Utils.h"
+#include "glow/IR/IR.h"
+#include "glow/Support/Debug.h"
+
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/ADT/SmallVector.h"
+#include "llvm/ADT/StringRef.h"
+#include "llvm/ADT/StringSet.h"
+#include "llvm/ADT/ilist.h"
+#include "llvm/ADT/ilist_node.h"
+
+#include <unordered_map>
+#include <unordered_set>
```

Comment:
could you please clean up unnecessary headers?

Reply:
Can I clean up unused headers in the .cpp too? For example, `ArrayRef`, `SmallVector`, `StringRef` and `StringSet` are not used anywhere in the source. I can leave debugging headers in for convenience.

Reply:
Sure, feel free remove any include directives that are not really used.